infra/resources/_modules/function_apps/locals.tf

```diff
@@ -36,7 +36,7 @@ locals {
       PidIssuerHealthCheckEnabled   = true
 
-      RevocationQueueName = var.revocation_queue_name
+      WalletInstanceRevocationQueueName = var.revocation_queue_name
```

Comment:
deletion should usually be done after deployment and switching to the new variable to avoid inconsistencies between the function deployment and the infra deployment. For this PR, both should be left and post-deployment an ad-hoc PR should be done to eliminate unused variables.